tests/fv3_conf/cpld_datm_cfsr.IN

```diff
@@ -0,0 +1,26 @@
+mkdir INPUT DATM_INPUT RESTART history MOM6_OUTPUT
+
+export DATM=true
+# FV3 fixed input
+export IATM=1760
+export JATM=880
+cp    @[RTPWD]/DATM/cfsr.SCRIP.nc DATM_INPUT
+ln -s  @[RTPWD]/DATM/CFSR/201110/* DATM_INPUT
+
+# MOM6 fixed input
+cp    @[RTPWD]/MOM6_FIX/@[OCNRES]/* ./INPUT
+cp    @[RTPWD]/MOM6_FIX_DATM/@[OCNRES]/* ./INPUT
+
+# CICE fixed input
+cp    @[RTPWD]/CICE_FIX/@[OCNRES]/grid_cice_NEMS_mx@[OCNRES].nc .
+cp    @[RTPWD]/CICE_FIX/@[OCNRES]/kmtu_cice_NEMS_mx@[OCNRES].nc .
+cp    @[RTPWD]/CICE_FIX/@[OCNRES]/mesh.mx@[OCNRES].nc .
+
+
+if [[ $OCNRES == '025' ]]; then
+cp    @[RTPWD]/MOM6_IC_DATM/025/2011100100/MOM*.nc ./INPUT
+cp    @[RTPWD]/CICE_IC_DATM/025/cice5_model_0.25.cpc*.nc cice_model.res.nc
+else
+cp    @[RTPWD]/MOM6_IC_DATM/100/2011100100/MOM6_IC_TS*.nc ./INPUT/MOM6_IC_TS.nc
+cp    @[RTPWD]/CICE_IC_DATM/100/cice5_model_1.00.cpc*.nc cice_model.res.nc
```

Comment:
Is there a reason that the DATM MOM6 and CICE ICs can't be in the same directory as used for the coupled model? 

Also, we should remove the hard-wired resolution identifiers in the directory path names:

`@[RTPWD]/CICE_IC/@[OCNRES]/cice_model_@[ICERES].res_2016100300.nc ./cice_model.res.nc`

Reply:
Done.

Reply:
This still looks like it is using MOM6_IC_DATM and CICE_IC_DATM. The file MOM6_IC_TS.nc is identical to that used by the coupled model so there is no reason to have a duplicate file. For the mx025, we do use the benchmark IC, but it can be placed in the same MOM6_IC directory used by the coupled model but in a subdirectory: RTPWD/MOM6_IC/2011100100

The CICE_ICs can be placed in the same directories the coupled model uses (025/050/100) since they have a date in the filename.

Reply:
Will we have an issue and future PR to use the same MOM6 IC and CICE IC directories as the coupled model?

Reply:
For some reason the removal of the DATM specific input directories doesn't show up in the PR differences but I see it when just looking directly at your branch. Change looks good! Thanks.